src/FTPObject.jl

```diff
@@ -104,7 +104,9 @@ Upload IO object "file" to the FTP server and save as "file_name".
 If "file" is not specified, the file "file_name" is uploaded.
 """ ->
 function upload(ftp::FTP, file_name::AbstractString, file=nothing)
```

Comment:
If my memory serves me correctly FTP clients usually name things in terms of "remote" or "local". Probably the `upload` function would have the following methods:
```julia
upload(ftp::FTP, local::AbstractString)
upload(ftp::FTP, local::AbstractString, remote::AbstractString)
upload(ftp::FTP, local::IO)
upload(ftp::FTP, local::IO, remote::AbstractString)
```